### PR TITLE
makes the original sqs message easier to access

### PR DESCRIPTION
**Pull request by curtis-rainforestqa**
this looks into exposing the original sqs message to the event processors. to make it easier to reference data on the original message object.

### CODE REVIEW

**Review by marianafranco — APPROVED** [review]
LGTM

File: lib/simple_sqs/processor.rb [hunk 1]
```diff
@@ -6,26 +6,26 @@
 require 'logger'
 
 class SimpleSqs::Processor
-  def process_sqs_message sqs_message
+  def process_sqs_message json_message_body, sqs_message = nil
```
Comment (shosti) [inline lib/simple_sqs/processor.rb:9]: Is this method generally used directly by outside users of the lib? If not, I think you could make the second arg non-optional so it isn't accidentally left out.

**Review by shosti — APPROVED** [review]
One question, but looks fine I think.

File: lib/simple_sqs/processor.rb (hunk 1 above)
Comment (curtis-rainforestqa) [reply lib/simple_sqs/processor.rb:9]: @shosti  `process_sqs_message` might be used by outsiders. we use it directly in a couple of our spout specs.

**Pull request merged**